ONSdigital/dp-permissions-api
# Update http handlers to use standardised success & error response types

### What
Service endpoints to use standardised success and error responses
Add default 500 error responses to prevent leaking any internal information.

### How to review
- Confirm http handlers implemented as standardised `baseHandler` type 
- Check code against coding standards, passing tests
- Changes may be easier to review by walking through the commits

### Who can review
Anyone
